lib/release_notes_bot/projects.ex

```diff
@@ -48,12 +48,21 @@ defmodule ReleaseNotesBot.Projects do
     %{client: client_name}
   end
 
-  # Parse the response from the first modal
   defp parse_inner_response(%{"client-select" => input}) do
-    # Populate a static select for all projects under client id in here
-    input["static_select-action"]["selected_option"]["value"]
-    |> String.to_integer()
-    |> ReleaseNotesBot.Clients.get_projects()
+    # Slack Channel ID is encoded in the only key. We need to decode it.
+    # It looks like this: "static_select-action:SOME-ID".
+    # This key is used to update the Channels table so that way a client_id
+    # is always associated with a particular channel.
+    [key] = Map.keys(input)
```

Comment:
So the first key is always the ID? I wonder if there's an elixir idiom to grab values by partial key match.